packages/apps/docs/src/components/Layout/components/Menu/menu.css.ts

```diff
@@ -1,19 +1,21 @@
-import { breakpoints, sprinkles } from '@kadena/react-ui/theme';
+import { breakpoints, sprinkles, vars } from '@kadena/react-ui/theme';
 
 import { $$leftSideWidth, $$sideMenu } from '../../global.css';
 
 import { style, styleVariants } from '@vanilla-extract/css';
 
 export const menuClass = style([
   sprinkles({
-    position: 'absolute',
-    paddingBottom: '$40',
+    position: 'fixed',
     height: '100%',
     width: '100%',
     background: '$background',
     overflow: 'hidden',
   }),
   {
+    top: vars.sizes.$17,
```

Comment:
these are not sprinkles?

Reply:
yes, I think so! but I'll double check now

Reply:
sorry! my bad! it's part of sprinkles. updated it